<dec f='glibc_src_2.29/sunrpc/rpc/clnt.h' l='107' type='enum auth_stat'/>
<offset>0</offset>
<doc f='glibc_src_2.29/sunrpc/rpc/clnt.h' l='107'>/* why the auth error occurred */</doc>
<use f='glibc_src_2.29/sunrpc/clnt_perr.c' l='97' macro='1' u='r' c='__GI_clnt_sperror'/>
<use f='glibc_src_2.29/sunrpc/clnt_perr.c' l='103' macro='1' u='r' c='__GI_clnt_sperror'/>
<use f='glibc_src_2.29/sunrpc/clnt_tcp.c' l='307' macro='1' u='w' c='clnttcp_call'/>
<use f='glibc_src_2.29/sunrpc/clnt_udp.c' l='492' macro='1' u='w' c='clntudp_call'/>
<use f='glibc_src_2.29/sunrpc/clnt_unix.c' l='284' macro='1' u='w' c='clntunix_call'/>
<use f='glibc_src_2.29/sunrpc/rpc_prot.c' l='265' macro='1' u='w' c='__GI__seterr_reply'/>
